src/PhpSpreadsheet/Worksheet/SheetView.php

```diff
@@ -115,6 +125,24 @@ public function setZoomScaleNormal($pValue)
         return $this;
     }
 
+    /**
+     * Set ShowZeroes setting.
+     *
+     * @param bool $pValue
+     */
+    public function setShowZeros($pValue)
+    {
+        $this->showZeros = $pValue;
+    }
+
+    /**
+     * @return bool
+     */
+    public function isShowZeros()
```

Comment:
While the use of `isShowZeroes()` is semantically cleaner; for historic reasons, all equivalent methods use `get`, so the use of `getShowZeros()` is better from the perspective of consistency within the library